src/lib/definitions.js

```diff
@@ -975,6 +976,7 @@ function getFHPaddingEntries(index)
             define('(RP_0_S + self)[10]', DOMWINDOW),
             define('(RP_4_A + Date())[30]', GMT),
             define('(RP_0_S + Audio)[11]', HTMLAUDIOELEMENT),
+            define('navigator.userAgent[0]', MOZILLA),
```

Comment:
Unknown identifiers don't work in definitions. The best thing to do is adding a constant definition for `navigator.userAgent` like this one for `location.constructor`:

https://github.com/fasttime/JScrewIt/blob/56a9d6afdc96d1ba3df0826b9bd41935ac7781b8/src/lib/definitions.js#L1542-L1546

That constant could be used in an expression like `'USER_AGENT[0]'`.